src/R/Support/Impl/Help/Packages/PackageInfo.cs

```diff
@@ -80,7 +80,7 @@ internal sealed class PackageInfo : NamedItemInfo, IPackageInfo {
                 try {
                     var result = await _host.Session.PackagesFunctionsNamesAsync(Name, REvaluationKind.BaseEnv);
                     functions = result.Children<JValue>().Select(v => (string)v.Value);
-                } catch (TaskCanceledException) { } catch (REvaluationException) { }
+                } catch (RException) { }
```

Comment:
No, but there can be side effects:
- `OperationCanceledException` may be required by tasks. For example:
```cs
var cts = new System.Threading.CancellationTokenSource();
var t = Task.Run(async () => {
    while (true) {
        await Task.Delay(1000, cts.Token);
    }
});
```
Task `t` will end when `cts.Cancel()` is called. If we wrap `await Task.Delay(1000, cts.Token);`:
```cs
var cts = new System.Threading.CancellationTokenSource();
var t = Task.Run(async () => {
    while (true) {
        try {
            await Task.Delay(1000, cts.Token);
        } catch (OperationCanceledException) { }
    }
});
```
task will be running forever.

- `RHostDisconnectedException` is captured on the ViewModel layer to show error messages. Swallowing it on the lower levels will break this code.